bin/php-language-server.php

```diff
@@ -10,6 +12,21 @@
     }
 }
 
-$server = new LanguageServer(new ProtocolStreamReader(STDIN), new ProtocolStreamWriter(STDOUT));
+if (count($argv) >= 3 && $argv[1] == '--tcp') {
```

Comment:
Ok. There's also still the JS side that needs to be updated to actually use this connection type.